Sources/TuistCore/Utils/DefaultConfigurationFetcher.swift

```diff
@@ -7,7 +7,7 @@ import XcodeGraph
 public protocol DefaultConfigurationFetching {
     func fetch(
         configuration: String?,
-        config: TuistCore.Config,
+        config: TuistCore.Tuist,
```

Comment:
What do you think about renaming the name of the arguments from `config` to `tuist`? Would you do it as part of this PR?